client/src/services/credentialsApi.js

```diff
@@ -1,6 +1,60 @@
+import jquery from 'jquery';
+
 class CredentialsApi {
-  static getCredentials() {
-    return fetch(process.env.REACT_APP_CREDENTIALS_API).then(response => {
+  static addCredential(data = {}) {
+    return fetch(process.env.REACT_APP_CREDENTIALS_API, {
```

Comment:
Wondering if we should have a wrapper for process.env.<API>. 
`ApiServer.CREDENTIALS_API` would read nicer.